docs/user/developer-onboarding.md

```diff
@@ -0,0 +1,118 @@
+# Developer Onboarding
+
+> **Audience**: This guide is for developers joining a repository that already has erk configured. If you're a project maintainer setting up erk for the first time, see [Project Setup](project-setup.md) instead.
+
+## Prerequisites
+
+Before you begin, ensure you have:
+
+- **erk CLI installed** - Follow the [erk installation guide](../../README.md) if needed
+- **Claude Code** - The AI-powered CLI that erk extends
+
+## Step 1: Clone the Repository
+
+Clone the repository as usual:
+
+```bash
+git clone <repo-url>
+cd <repo-name>
+```
+
+Since erk is already configured by the project maintainer, you'll find:
+
+- **`erk.toml`** - Project configuration
+- **`.claude/`** - Kit artifacts (commands, skills, hooks)
+
+These are committed to git and ready to use.
+
+## Step 2: Verify Your Setup
+
+Check that erk recognizes the project configuration:
+
+```bash
+erk status
+```
+
+You should see your project configuration displayed.
```

Comment:
setting up the shell is the hardest thing and needs to be done on a per-developer.  
  
also user should be using erk doctor in this